<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="plan_file">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>153</width>
      <height>94</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="planfile_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Plan File #</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="planfile_entry">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="planfile_Btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="select_folder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="open_folder_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>84</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Active Category</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pump_checkB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pump</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pressure_checkB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pressure</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="gravity_checkB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gravity</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="options_pane">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>141</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="options_Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>42</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Options</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_existing_Btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Existing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create_new_Btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Create New</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create_pdf">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Create PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_btn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>230</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset All Fields</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="work_area">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>153</width>
      <height>94</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="work_entry">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Work Area</string>
     </property>
    </widget>
    <widget class="QPushButton" name="work_entry_Btn">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Enter</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="DorP_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>146</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="DorP_label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>106</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Development or CIP</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="development_checkB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>94</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Development</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="CIP_checkB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>CIP</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>1191</width>
      <height>681</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="select_folder_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>125</width>
      <height>110</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="open_folder_label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>84</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Folders Found</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pump_folder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pump</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pressure_folder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Pressure</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="gravity_folder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>75</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gravity</string>
     </property>
    </widget>
    <widget class="QFrame" name="select_folder_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>125</width>
       <height>110</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="open_folder_label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>84</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Active Category</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pump_checkB_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pump</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pressure_checkB_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Pressure</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="gravity_checkB_4">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>75</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gravity</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>work_entry</tabstop>
  <tabstop>work_entry_Btn</tabstop>
  <tabstop>planfile_entry</tabstop>
  <tabstop>planfile_Btn</tabstop>
  <tabstop>pump_checkB</tabstop>
  <tabstop>pressure_checkB</tabstop>
  <tabstop>gravity_checkB</tabstop>
  <tabstop>open_existing_Btn</tabstop>
  <tabstop>create_new_Btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
